pkg/server/api/svid/v1/service.go

```diff
@@ -25,14 +30,65 @@ type Service interface {
 
 // New creates a new SVID service
 func New(config Config) Service {
-	return &service{}
+	return &service{
+		ServerCA:    config.ServerCA,
+		TrustDomain: config.TrustDomain,
+	}
 }
 
 type service struct {
+	ServerCA    ca.ServerCA
+	TrustDomain spiffeid.TrustDomain
```

Comment:
These fields should probably not be exported? Also, we can use more terse names for the unexported fields if you want without sacrificing readability, IMO :)
```suggestion
	ca    ca.ServerCA
	td spiffeid.TrustDomain
```

Reply:
👍